internal/server/handlers.go

```diff
@@ -329,7 +329,8 @@ func (a *API) CreateToken(c *gin.Context, r *api.EmptyRequest) (*api.CreateToken
 	if access.AuthenticatedIdentity(c) != nil {
 		err := a.UpdateIdentityInfoFromProvider(c)
 		if err != nil {
-			return nil, fmt.Errorf("%w: update ident info from provider: %s", internal.ErrForbidden, err)
+			// TODO: why would this fail? seems like this should be a 5xx error
+			return nil, fmt.Errorf("update ident info from provider: %w", err)
```

Comment:
From my reading any error here would likely be a runtime error like the database or provider being unavailable because of a network problem or outage. That seems like it should be a 5xx error. Are there other errors that I've missed?